packages/flutter_test/lib/src/binding.dart

```diff
@@ -1218,13 +1218,15 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// which can cause differences in behavior.
   fullyLive,
 
-  /// Ignore any request to schedule a frame.
+  /// Ignore any request to schedule a frame from the framework or
+  /// [WidgetTester].
```

Comment:
We contridict this in the next paragarph when we say you can use `WidgetTester.pumpBenchmark`. Maybe you meant to name specific methods?

Reply:
`[TestWidgetsFlutterBinding.pump]` is better here I believe.